include/hpp/constraints/implicit.hh

```diff
@@ -120,6 +120,20 @@ class HPP_CONSTRAINTS_DLLAPI Implicit {
  public:
   /// Operator equality
   bool operator==(const Implicit& other) const;
+  /// Operator assignation
+  Implicit& operator=(const Implicit& other) {
+    comparison_ = other.comparison_;
+    rhs_ = other.rhs_;
+    parameterSize_ = other.parameterSize_;
+    function_ = other.function_;
+    rhsFunction_ = other.rhsFunction_;
+    mask_ = other.mask_;
+    activeRows_ = other.activeRows_;
+    inactiveRows_ = other.inactiveRows_;
+    inequalityIndices_ = other.inequalityIndices_;
+    equalityIndices_ = other.equalityIndices_;
+    return *this;
+  }
```

Comment:
Out of curiosity, why do we need this ?

Reply:
According to gcc, the implicitly defined `operator=` with an explicitly defined copy constructor is deprecated.

Reply:
ref. https://en.cppreference.com/w/cpp/language/rule_of_three